frontend/src/metabase/actions/containers/ActionCreator/tests/ActionCreator-QueryActions.unit.spec.ts

```diff
@@ -0,0 +1,123 @@
+import nock from "nock";
+
+import { screen, getIcon, queryIcon } from "__support__/ui";
+
+import {
+  createMockActionParameter,
+  createMockQueryAction,
+} from "metabase-types/api/mocks";
+
+import { setup } from "./common";
+
+describe("ActionCreator > Query Actions", () => {
+  afterEach(() => {
+    nock.cleanAll();
+  });
+
+  describe("new action", () => {
+    it("renders correctly", async () => {
+      await setup();
+
+      expect(screen.getByText(/New action/i)).toBeInTheDocument();
```

Comment:
Are we asserting this UI in the screenshot?
<img width="687" alt="Screenshot 2023-02-17 at 11 31 39 AM" src="https://user-images.githubusercontent.com/1937582/219640965-d19220fd-41c7-4b4f-b63c-81d7bbd66851.png">

Can we just use a plain string though?

```suggestion
      expect(screen.getByText("New Action")).toBeInTheDocument();
```